campaignion_email_to_target/campaignion_email_to_target.info

```diff
@@ -5,6 +5,7 @@ package = Campaignion
 
 dependencies[] = campaignion_action
 dependencies[] = campaignion_vue
+dependencies[] = campaignion_webform_token
```

Comment:
Why is this dependency added?

Reply:
ah, yes, not needed any more.
I used to rely on `_campaignion_webform_tokens_get_submission` before realising that it's already in the token data... :innocent: